src/components/organisms/rma-select-product-table/index.tsx

```diff
@@ -0,0 +1,262 @@
+      ...toReturn,
+      [id]: {
+        ...toReturn[id],
+        reason: reason,
+        note: note,
+      },
+    }
+
+    setToReturn(newReturns)
+  }
+
+  const isLineItemCanceled = (item) => {
+    const { swap_id, claim_order_id } = item
+    const travFind = (col, id) =>
+      col.filter((f) => f.id == id && f.canceled_at).length > 0
+
+    if (swap_id) {
+      return travFind(order.swaps, swap_id)
+    }
+    if (claim_order_id) {
+      return travFind(order.claims, claim_order_id)
+    }
+    return false
+  }
+
+  console.log(toReturn)
+
+  return (
+    <Table>
+      <Table.HeadRow className="text-grey-50 inter-small-semibold">
+        <Table.HeadCell colspan={2}>Product Details</Table.HeadCell>
+        <Table.HeadCell className="text-right pr-8">Quantity</Table.HeadCell>
+        <Table.HeadCell className="text-right">Refundable</Table.HeadCell>
+        <Table.HeadCell></Table.HeadCell>
+      </Table.HeadRow>
+      <Table.Body>
+        {allItems.map((item) => {
+          // Only show items that have not been returned,
+          // and aren't canceled
+          if (
+            item.returned_quantity === item.quantity ||
+            isLineItemCanceled(item)
+          ) {
+            return
+          }
+          const checked = item.id in toReturn
+          return (
+            <>
+              <Table.Row className={clsx("border-b-grey-0 hover:bg-grey-0")}>
+                <Table.Cell>
+                  <div className="items-center ml-1 h-full flex">
+                    <div
+                      onClick={() => handleReturnToggle(item)}
+                      className={`w-5 h-5 flex justify-center text-grey-0 border-grey-30 border cursor-pointer rounded-base ${
+                        checked && "bg-violet-60"
+                      }`}
+                    >
+                      <span className="self-center">
+                        {checked && <CheckIcon size={16} />}
+                      </span>
+                    </div>
+
+                    <input
+                      className="hidden"
+                      checked={checked}
+                      tabIndex={-1}
+                      onChange={() => handleReturnToggle(item)}
+                      type="checkbox"
+                    />
+                  </div>
+                </Table.Cell>
+                <Table.Cell>
+                  <div className="min-w-[240px] flex py-2">
+                    <div className="w-[30px] h-[40px] ">
+                      <img
+                        className="h-full w-full object-cover rounded"
+                        src={item.thumbnail}
+                      />
+                    </div>
+                    <div className="inter-small-regular text-grey-50 flex flex-col ml-4">
+                      <span>
+                        <span className="text-grey-90">{item.title}</span> test
+                      </span>
+                      <span>{item.variant.title}</span>
```

Comment:
Variant is not necessarily present (it would be in almost all cases, but there is a couple of exceptions like gift cards)
```suggestion
                      <span>{item?.variant?.title || ""}</span>
```